### PR TITLE
fix RTP issue in PFH main Loop

### PR DESCRIPTION
**Pull request by jokoho48**

### CODE REVIEW

File: addons/common/fnc_removePerFrameHandler.sqf [hunk 1]
```diff
@@ -26,7 +26,9 @@ Author:
 
 params ["_publicHandle"];
 private "_handle";
-if (isNil "_publicHandle" || (_publicHandle < 0)) exitWith {}; // Nil/no handle, nil action
+if ((isNil "_publicHandle" || (_publicHandle < 0)) && {(count GVAR(PFHhandles)) < _publicHandle}) exitWith { // Nil/no handle, nil action or handle is larger than Public Handle Array
+    WARNING("PFH ID allready Removed or handle Not Exist or handle is smaller than 0");
```
Comment (jonpas) [inline addons/common/fnc_removePerFrameHandler.sqf]: PFH ID already removed or handle does not exist or handle is smaller than 0

Comment (MikeMatrix) [inline addons/common/fnc_removePerFrameHandler.sqf]: I'd go with `Invalid or not existing PFH ID.`

Comment (jonpas) [inline addons/common/fnc_removePerFrameHandler.sqf]: That's even better.

**Comment by jokoho48** [comment]
ok i and @jonpas tested it no issues found

**Comment by jonpas** [comment]
:+1:

File: addons/common/fnc_removePerFrameHandler.sqf [hunk 2]
```diff
@@ -25,8 +25,12 @@ Author:
 #include "script_component.hpp"
 
 params ["_publicHandle"];
+
+if (isNil "_publicHandle" || {(_publicHandle < 0)} || {!((count GVAR(PFHhandles)) >= _publicHandle)}) exitWith { // Nil/no handle, nil action or handle is larger than Public Handle Array
```
Comment (PabstMirror) [inline addons/common/fnc_removePerFrameHandler.sqf]: Should this be `{(count GVAR(PFHhandles) <= _publicHandle)}`?

If the count is 0, and the _publicHandle is 0, you want it to be true and exit.

**Comment by ViperMaul** [comment]
@PabstMirror 
Let's make a new pull request and make that change. This PR is already merged.

**Pull request merged**